install_script.sh.template

```diff
@@ -445,15 +445,44 @@ if [ -n "$DD_AGENT_FLAVOR" ]; then
     agent_flavor=$DD_AGENT_FLAVOR #Eg: datadog-iot-agent
 fi
 
-declare -A flavor_to_readable
-flavor_to_readable=(
-    ["datadog-agent"]="Datadog Agent"
-    ["datadog-iot-agent"]="Datadog IoT Agent"
-    ["datadog-dogstatsd"]="Datadog Dogstatsd"
-    ["datadog-fips-proxy"]="Datadog FIPS Proxy"
-    ["datadog-heroku-agent"]="Datadog Heroku Agent"
-)
-nice_flavor=${flavor_to_readable[$agent_flavor]}
+# Emulate hashmap with simple switch case
+getMapData() {
+
+    if [ "$1" = "flavor_to_readable" ]; then
+        case "$2" in
+            "datadog_agent")
+                DATA="Datadog Agent"
+                ;;
+             "datadog_iot_agent")
+                DATA="Datadog IoT Agent"
+                ;;
+             "datadog_dogstatsd")
+                DATA="Datadog Dogstatsd"
+                ;;
+             "datadog_fips_proxy")
+                DATA="Datadog FIPS Proxy"
+                ;;
+             "datadog_heroku_agent")
+                DATA="Datadog Heroku Agent"
+                ;;
+              *)
+                DATA="Unknown"
+                ;;
+         esac
+    elif [ "$1" = "flavor_to_system_service" ] && [ "$2" = "datadog_dogstatsd" ]; then
+        DATA="datadog-dogstatsd"
+    elif [ "$1" = "flavor_to_etcdir" ] && [ "$2" = "datadog_dogstatsd" ]; then
+        DATA="/etc/datadog-dogstatsd"
+    elif [ "$1" = "flavor_to_config" ] && [ "$2" = "datadog_dogstatsd" ]; then
+        DATA="$etcdir/dogstatsd.yaml"
+    fi
+    printf '%s' "$DATA"
+}
+
+agent_flavor_formatted=$(echo "$agent_flavor" | tr "-" "_")
```

Comment:
Is there a reason for this change? Shell options convention is more with dashes (whereas variables are more with underscore), so I'm not sure we should change this

Reply:
Necessary no, just because otherwise we would have in ``getMapData`` Map names with underscore like ``some_map_name`` and flavor like ``some-flavor`` in the if else switch statement. So purely to improve readability if you think it's useless I'll remove it wdyt ?

Reply:
indeed, because $1 is the old variable name and $2 the option name. I was just surprised because in your PR there is a change from `-` to `_` internally in the option name and I didn't see immediately this line, I guess it would be less confusing in the future. I'm ok with both, I let you chose :)